src/config.c

```diff
@@ -862,6 +934,50 @@ void RSConfigExternalTrigger_Register(RSConfigExternalTrigger trigger, const cha
   RSGlobalConfigTriggers[numTriggers++] = trigger;
 }
 
+#ifdef MT_BUILD
+
+void UpgradeDeprecatedMTConfigs() {
+  RSConfigVar *workers = findConfigVar(&RSGlobalConfigOptions, "WORKERS");
+  RSConfigVar *minOperationWorkers = findConfigVar(&RSGlobalConfigOptions, "MIN_OPERATION_WORKERS");
+  if (workers->flags & RSCONFIGVAR_F_MODIFIED || minOperationWorkers->flags & RSCONFIGVAR_F_MODIFIED) {
+    return; // New configurations were set. Ignore the deprecated ones.
+  }
+  RSConfigVar *mtMode = findConfigVar(&RSGlobalConfigOptions, "MT_MODE");
+  RSConfigVar *workerThreads = findConfigVar(&RSGlobalConfigOptions, "WORKER_THREADS");
+  if (!(mtMode->flags & RSCONFIGVAR_F_MODIFIED) && !(workerThreads->flags & RSCONFIGVAR_F_MODIFIED)) {
+    return; // No deprecated configurations were set.
+  }
+
+  // We now know that deprecated configurations were set, and new configurations were not set.
+  if ((mt_mode_config == MT_MODE_OFF && numWorkerThreads_config != 0) ||
```

Comment:
`mt_mode_config == MT_MODE_OFF && numWorkerThreads_config != 0` was not considered as an error in 2.8

Reply:
Are you sure? I’ll check again.
Anyway it means nothing. What will be the right translation here? OFF or set the number?

Reply:
Yes I checked. But I guess you are right, nothing to do here